.travis.yml

```diff
@@ -1,24 +1,12 @@
-branches:
-  only:
-    - master
```

Comment:
I used this to tag releases and launch deploys to Bintray...how come we're removing this?

Reply:
Ah, so it deploys a snapshot if its master and a release version if its a tag? I removed it because I thought it made since to have Travis builds run on pull requests, which it seems like presently it is preventing that from happening

Reply:
ah interesting.  i was trying to use the deploy section below to only upload on release tagged branches (https://docs.travis-ci.com/user/customizing-the-build/#Deploying-your-Code).  is there a way to build on pull requests and still have this option?

Reply:
So we could do something in a script like:
```
# Check if tag is not empty
# http://stackoverflow.com/a/21407325/895797
if [ -n "$TRAVIS_TAG" ]; then
     echo "Deploying project";
     eval "./gradlew deploy"
else
     echo "Building project";
     eval "./gradlew build"
fi
```
and we would just run this `.sh` script in Travis

Reply:
cool I like it =).  thanks =)

Reply:
did you add yet?